JSDemos/Demos/TreeList/ColumnChooser/description.md

```diff
@@ -1 +1,13 @@
-The Column Chooser is&nbsp;an&nbsp;element that allows a&nbsp;user to&nbsp;alter the set of&nbsp;visible columns at&nbsp;runtime. To&nbsp;open the column chooser, click the _&laquo;Column Chooser&raquo;_ button at&nbsp;the top of&nbsp;TreeList. Now, you can drag and drop column headers to&nbsp;and from the TreeList, thus showing or&nbsp;hiding columns. Use the column chooser&rsquo;s search box to&nbsp;quickly find the required column.
\ No newline at end of file
+The column chooser allows users to hide and display columns at runtime. To enable it, assign **true** to the **columnChooser**.[enabled](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columnChooser/#enabled) property. 
```

Comment:
"Hide and display" - in this phrase "display" doesn't look like an opposite to hide. "Hide" implies a change of state from visible to invisible. "Display" doesn't really mean the columns were previously hidden. If a control displays something, it doesn't mean this something was hidden before. So I'd maybe use "change visibility" or "hide and make visible". Even "hide and show" is better, I guess. 

And then I think the line about columns that can always remain visible works better here (I moved it up from the very end.)

```suggestion
If you want users to change column visibility at runtime, enable the Column Chooser. Set the **columnChooser**.[enabled](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columnChooser/#enabled) property to **true**.  Note that you can make sure that certain columns always stay visible. To do so, set the **columns[]**.[allowHiding](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columns/#allowHiding) property to **false**. 
```